src/styles/claudy.css

```diff
@@ -0,0 +1,34 @@
+[role=banner] .coz-claudy {
+  position: fixed;
+  bottom: 5em;
+  right: 2em;
+}
+
+@media (min-width: 64em) {
+  [role=banner] .coz-claudy {
+    bottom: 2em;
+  }
+}
+
+[role=banner] .coz-claudy .coz-claudy-icon {
+  width: 3.5em;
+  height: 3.5em;
+  border-radius: 100%;
+  border: none;
+  background-color: var(--dodger-blue);
+  background-image: url('../assets/icons/16/icon-claudy.svg');
```

Comment:
You could have write the shorthand here, like this:
`background: var(--dodger-blue) url('../assets/icons/16/icon-claudy.svg') .5em / 2.5em no-repeat;`